Digital-launcher.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec "/app/lib/python" "-OO" "/app/Digital.py" "$@"
```

Comment:
If `Digital.py` is directly executable then just put it in `/app/bin` and remove the need for a launcher?